### PR TITLE
Adding initial REST Framework; Adding method to get user stats

### PR DESCRIPTION
**Pull request by bstascavage**
Right now there isn't a way for a developer to get information on the bot while the bot is running.  Sure, you can check information by logging into the app or debugging, but there is no way to query the bot for various statistics.  I added a REST Framework to the bot so that a user can retrieve this information by accessing various endpoints.

Some use cases:

- Developers trying to gather information/test methods

- Developers wanting to write other applications that interact with the bot, such as web views or chat commands

- A way to access the bot without logging in; very useful for people with multiple accounts (or a bot test account)

When the bot is started, it starts to listen on port 5000 for any requests.  This runs in parallel to the main bot via a fork.  As of this PR, there is only one method (more will be added once the Framework is approved):

`/api/player_info`  - Returns the results of the `bot.get_player_info()` method

Example:

`bstascav@kefka:~/pgoapi# curl -XGET localhost:5000/api/player_info
{
    "Level": 7,
    "Experience": 26465,
    "Experience until next level": 1535,
    "Pokemon Captured": 73,
    "Pokestops Visited": 34
}`

### CODE REVIEW

**Comment by bstascavage** [comment]
FYI, as of typing this, your Slack request link isn't sending emails.  I am not sure who to contact about that.

File: pokemongo_bot/__init__.py [hunk 1]
```diff
@@ -266,14 +270,19 @@ def get_player_info(self):
...
-                                        print('[#] -- Pokemon Captured: {pokemons_captured}'.format(**playerdata))
+                                        player_stats['Pokemon Captured'] = playerdata['pokemons_captured']
 
                                     if 'poke_stop_visits' in playerdata:
-                                        print('[#] -- Pokestops Visited: {poke_stop_visits}'.format(**playerdata))
+                                        player_stats['Pokestops Visited'] = playerdata['poke_stop_visits']
+
+        if print_stats:
+            for key in player_stats:
+                 print("[#] -- %s: %s" % (key, player_stats[key]))
```
Comment (jtdroste) [inline pokemongo_bot/__init__.py]: I would change this to ```print("[#] -- {}: {}".format(key, player_stats[key]))``` - to keep consistency

**Comment by jtdroste** [comment]
I'll look into the slack thing shortly.  Thanks!

**Comment by bstascavage** [comment]
@jtdroste Updated the print statement as per your recommendation

**Comment by jtdroste** [comment]
Thanks @bstascavage!  On the slack issue, it seems we're being rate limited due to the amount of invites being requested.  I'll try to get you in there soon.

**Comment by bstascavage** [comment]
Thanks @jtdroste!  Hopefully it will be soon!

**Pull request merged**